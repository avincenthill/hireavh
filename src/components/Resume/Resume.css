.resume-link {
  flex-direction: column;
  align-items: center;
  justify-content: center;
  color: var(--c-1);
}
