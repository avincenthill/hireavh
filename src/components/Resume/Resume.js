import React, { Component } from "react";
import { Document, Page } from "react-pdf";
import resume from "../../assets/pdf/avh_resume.pdf";
import { pdfjs } from "react-pdf";
import Header from "../Header/Header";
import Footer from "../Footer/Footer";
import content from "../../content/content";
import "./Resume.css";

export default class Resume extends Component {
  componentDidMount() {
    pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/2.1.266/pdf.worker.js`;
  }

  render() {
    return (
      <div>
        <Header></Header>
        <div>
          <Document className="resume-document" file={resume}>
            <a
              className="resume-link"
              href={require("../../assets/pdf/avh_resume.pdf")}
              download={content.resume.downloadName}
            >
              <Page pageNumber={1} height={window.screen.height * 0.5} />
              <h3 className="resume-cta">{content.resume.cta}</h3>
            </a>
          </Document>
        </div>
        <Footer></Footer>
      </div>
    );
  }
}
