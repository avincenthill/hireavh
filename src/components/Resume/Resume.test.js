import React from 'react';

import Resume from './Resume';
import renderer from 'react-test-renderer';

describe('Resume', () => {
  describe('snapshot', () => {
    it('should match the last saved snapshot', () => {
      const tree = renderer.create(<Resume />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
