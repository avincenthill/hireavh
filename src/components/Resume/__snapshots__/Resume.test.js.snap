// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume snapshot should match the last saved snapshot 1`] = `
<div>
  <div>
    <div>
      <div
        className="nav nav-top"
      >
        <div
          className="nav-icon $
          this.props.isBottom ? \\"nav-icon-bottom\\" : \\"nav-icon-top\\"
        }"
        >
          <svg
            fill="currentColor"
            height="1em"
            onClick={[Function]}
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "var(--c-1)",
                "height": "3rem",
                "width": "3rem",
              }
            }
            viewBox="0 0 8 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8 4v1H0V4h8zM0 8h8V7H0v1zm0 3h8v-1H0v1z"
              fillRule="evenodd"
            />
          </svg>
        </div>
      </div>
      <a
        className="header-container"
        href="/about"
      >
        <div
          className="header-switch-theme"
        >
          <div
            className="toggle-switch"
          >
            <input
              className="toggle-switch-checkbox"
              defaultChecked={true}
              id="toggle-switch-theme"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              className="toggle-switch-label"
              htmlFor="toggle-switch-theme"
            >
              <span
                className="toggle-switch-inner"
                data-no="üåô"
                data-yes="‚òÄÔ∏è"
              />
            </label>
          </div>
        </div>
        üè† about
      </a>
    </div>
    <div
      className="page-container"
    >
      <div>
        <div
          className="react-pdf__Document resume-document"
        >
          <div
            className="react-pdf__message react-pdf__message--loading"
          >
            Loading PDF‚Ä¶
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        className="nav nav-bottom"
      >
        <div
          className="nav-icon $
          this.props.isBottom ? \\"nav-icon-bottom\\" : \\"nav-icon-top\\"
        }"
        >
          <svg
            fill="currentColor"
            height="1em"
            onClick={[Function]}
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "var(--c-1)",
                "height": "3rem",
                "width": "3rem",
              }
            }
            viewBox="0 0 8 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8 4v1H0V4h8zM0 8h8V7H0v1zm0 3h8v-1H0v1z"
              fillRule="evenodd"
            />
          </svg>
        </div>
      </div>
      <a
        className="footer-container"
        href="/about"
      >
        <span>
          üè† about
        </span>
      </a>
    </div>
  </div>
</div>
`;
